manifests/w/Winamp/Winamp/5.90.0.9999/Winamp.Winamp.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with Notepad
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: "Winamp.Winamp"
+PackageVersion: "5.90.0.9999"
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 6.1.7601.0
+InstallerType: "nullsoft"
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

Reply:
5.90.0.9999 folder is old and is not part of this PR.
No changes were made to any files in that old folder.

This PR is for new 5.91.0.10029 folder only.